src/test/java/benchmark/BenchmarkUtils.java

```diff
@@ -1,19 +1,32 @@
 package benchmark;
 
-import com.google.common.io.Files;
+import com.google.common.io.ByteStreams;
 
-import java.io.File;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
 import java.net.URL;
 import java.nio.charset.Charset;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.concurrent.Callable;
 
 public class BenchmarkUtils {
 
+    @SuppressWarnings("UnstableApiUsage")
     static String loadResource(String name) {
         return asRTE(() -> {
             URL resource = BenchmarkUtils.class.getClassLoader().getResource(name);
-            return String.join("\n", Files.readLines(new File(resource.toURI()), Charset.defaultCharset()));
-
+            if (resource == null) {
+                throw new IllegalArgumentException("missing resource: " + name);
+            }
+            byte[] bytes;
+            try (InputStream inputStream = resource.openStream()) {
+                // In GraphQL Java version 21 and above, this Guava helper is replaced with Java 9's readAllBytes()
+                bytes = ByteStreams.toByteArray(inputStream);
```

Comment:
Fun one: readAllBytes is only in Java 9 onwards so here's the Guava equivalent. We selectively shade in Guava files (e.g. Immutable objects already in use) so we're alright to use Guava here.